dd-trace/src/log.rs

```diff
@@ -139,10 +139,13 @@ pub fn print_log<I: Into<String>>(
     if lvl == crate::log::LevelFilter::Error {
         eprintln!("\x1b[91m{lvl}\x1b[0m {file}:{line} - {log}");
 
-        crate::telemetry::add_log_error(
-            template.unwrap_or(&log),
-            Some(format!("Error: {log}\n at {file}:{line}")),
-        );
+        if let Some(template) = template {
+            // there should be no dynamic messages in either log message or log stack_trace
```

Comment:
I find this comment a bit confusing. I think it would be better if it said what we do and why, ie. `// we should only send the template to telemetry to not leak sensitive information`.